configs/sot/siamese_rpn/README.md

```diff
@@ -43,3 +43,15 @@ The best model on LaSOT is submitted to [the evaluation server on TrackingNet Ch
 |    Backbone     |  Style  | Lr schd | Mem (GB) | Inf time (fps) | Success | Norm precision | Config | Download |
 | :-------------: | :-----: | :-----: | :------: | :------------: | :----: | :----: | :------: | :--------: |
 |    R-50    |  -  |   20e    |      | -              | 70.6 | 77.6 | [config](siamese_rpn_r50_1x_trackingnet.py) | [model](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20201218_051019-3c522eff.pth) &#124; [log](https://download.openmmlab.com/mmtracking/sot/siamese_rpn/siamese_rpn_r50_1x_lasot/siamese_rpn_r50_1x_lasot_20201218_051019.log.json) |
+
+### OTB100
+
+The checkpoints from 10-th to 20-th epoch will be evaluated during training.
+
+After training, you need to pick up the best checkpoint from the log file, then use the best checkpoint to search the hyperparameters on OTB100 following [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md#siameserpn-test-time-parameter-search) to achieve the best results.
+
+**Note** We can not reproduce the results reported in the official [SiameseRPN++](https://github.com/STVIR/pysot) codebase. In our codebase, the Success and Precision are lower 3.6 and 2.7 respectively than those of the official report.
```

Comment:
We need to point out that the official codebase pysot cann't train the same results in the paper.
Please also tell users that we only get xxx success and xxx precision by following the training and hyperparameters searching instructions of pysot, which is lower than ours by ...... points